spec.md

```diff
@@ -0,0 +1,238 @@
+
+As an example, [OCI Images][image-layer] are represented through an ordinal collection of compressed files. Each blob represents a layer. Each layer overlays the previous layer.
+
+Other artifacts may be represented as a single file, such as config artifact representing a deployment specification. Other artifacts may include a config blob, and a collection of binaries compressed as another blob. By separating the blob, the artifact author can benefit from layer de-duplication and concurrent downloading of each blob.
+
+### Layer File Format
+
+Layers are persisted as blobs within registries. The blobs can be a single file or a collection of files. The persistance format is up to the artifact author. They may choose to persist individual files with their native or custom format, such as `.config`, `.json`, `.bin`, `.sif`, and compress them with various formats such as `.gzip`.
+
+Large files can benefit from compression when being transferred across the network. However, decompression takes time and compute cycles. For smaller, individual files, the decompression may take longer than downloading the file in its original format.
+
+### Layer Versioning
+
+Layers MUST be versioned to future proof new enhancements that may evolve. How the artifact tooling processes forward and backward compatibility is up to the artifact and tooling authors.
+
+### Defining Layer Types
+
+Artifact layers are expected to be unique, per artifact author. A layer that represents an OCI Image is not expected to be shared with a Helm Chart. To differentiate layers, artifact authors SHOULD create unique artifact layer types.
+
+Artifact layer types follow the same convention as manifests, utilizing the `layer.mediaType` with the following format:  
+`application/vnd.`[org|company]`.`[layerType]`.`[layerSubType]`.layer.`[version]`+`[fileFormat]
+
+### Example Layer Types
+
+|Artifact Type|mediaType|
+|-|-|
+|OCI Image|`application/vnd.oci.image.layer.v1+tar`|
+|Docker Image|`application/vnd.docker.image.rootfs.diff.tar.gzip`|
+|Helm Chart|`application/vnd.cncf.helm.chart.layer.v1+tar`|
+|Singularity SIF Layer|`application/vnd.sylabs.sif.layer.v1+tar`|
+
+### One or More Layer Types
+
+> NOTE: Should this section be in the `spec.md`, or in an `artifact-authors.md` doc?
+
+The number of layer types is up to the artifact author. Some things to consider when designing the layer format:
+
+- **Continuos Builds:** will developers automate building and pushing the artifact to a registry? For each rebuild, what will change? Consider the docker layering model, where users benefit from base layers that don't change often. If only a small piece of content changes, can you separate that content into a unique layer?
```

Comment:
nit: "Continuos" -> "Continuous".

Reply:
I'm moving this to the [artifact authors section](https://github.com/opencontainers/artifacts/pull/6) as it's more in the opinion category.